### DISCUSSION TITLE
Aluguel de equipamentos PCs, celulares e afins

### DISCUSSION THREAD

**Discussion by anacsv** [anacsv/tech-br-em-pt#13] [Lojas]
Achei interessante a proposta, se servir para alguém tai ai uma das empresas que prestam esse serviço:

https://www.futurdata.pt/pt/aluguer/apple/

**Comment by jvitorinoj** [comment]
Uma alternativa ao aluguel, é comprar equipamentos usados porém com garantia em brechos como o [webuy.com ](webuy.com). 
Montei 2 setups em 12/23 (2 monitores 27,  2 teclados e um mouse logitech) por €260. 
Possui lojas fisicas em quase todas as grandes cidades de Portugal.

Outro similar é o http://www.portoalternativo.com/
